packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -564,7 +564,15 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
       return FlutterSkippingGoldenFileComparator(
         baseDirectory.uri,
         goldens,
-        'No network connection available for contacting Gold.',
+        'OSError was thrown retrieving baselines. There may not be a network '
+          'connection. Switching to FlutterSkippingGoldenFileComparator.',
+      );
+    } on io.SocketException catch (_) {
+      return FlutterSkippingGoldenFileComparator(
+        baseDirectory.uri,
+        goldens,
+        'SocketException was thrown retrieving baselines. There may not be a '
+          'network connection. Switching to FlutterSkippingGoldenFileComparator.',
```

Comment:
nit: I think this can also happen if there's just no route to the host or DNS lookup fails (which may be different than just not having a network connection) - and it can probably happen if Skia Gold is down for some reason. 

I'd lean towards not explaining why and just saying "SocketException occurred, could not reach Skia Gold" or something to that effect.

Reply:
Updated! :)